Sources/WebSocketKit/WebSocket.swift

```diff
@@ -126,15 +169,21 @@ public final class WebSocket {
     }
 
     func handle(incoming frame: WebSocketFrame) {
+        self.lastReadTime = .now()
```

Comment:
Would it make sense to reschedule the next ping (cancelling the currently scheduled one) whenever we update `lastReadTime`?

Reply:
It might also make sense to have this take writes into consideration. Maybe we call this `lastActivityTime` or something and update it in the `send` method?